modules/crm_core_match/crm_core_match.test.inc

```diff
@@ -18,148 +18,148 @@ function crm_core_match_testing_page_title($contact) {
  *
  * @return string
  */
-function crm_core_match_testing_page($contact) {
-  $output = '';
-  $engines = crm_core_match_get_engines();
+//function crm_core_match_testing_page($contact) {
```

Comment:
this could be a useful feature but should be re-introduced as an entity operation or so with an auto-complete to select a contact or something like that? Probably easier to drop and have a task/issue to add it again?